schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java

```diff
@@ -453,6 +453,219 @@ private static void repairAll(
     }
   }
 
+  // For backward compatibility.
```

Comment:
Maybe we don’t need this comment if we’re not planning to remove these methods in a future release. Or do we plan to?

Reply:
Addressed it in https://github.com/scalar-labs/scalardb/pull/2760/commits/e7b6082f33b1a769d760023ebc5b4b31c1be4456